gpbft/participant.go

```diff
@@ -89,6 +89,10 @@ func (p *Participant) CurrentRound() uint64 {
 	return p.gpbft.round
 }
 
+func (p *Participant) CurrentInstance() uint64 {
+	return p.currentInstance
```

Comment:
Since this method is externally invoked, we can't control when that is, so I think this needs protection by the mutex (like validation).